liquid.py

```diff
@@ -27,58 +27,63 @@
 config = configparser.ConfigParser()
 config.read('liquid.ini')
 
+
 if 'extra_jobs' in config:
     for job, path in config['extra_jobs'].items():
         JOBS.append((job, Path(path)))
 
 
-def get_config(env_key, ini_path, default):
-    value = os.environ.get(env_key)
-    if value is not None:
-        return value
+def get_config(env_key, ini_path, default=None):
+    if env_key and os.environ.get(env_key):
+        return os.environ[env_key]
+
+    if ini_path.endswith(':'):
```

Comment:
This adds a lot of complexity to `get_config`. It looks like the end goal is to get collections from `liquid.ini`, which can be achieved outside of this function, with another function e.g. like this:

```python
def get_collections():
    for section_name in config:
        if section_name.startswith('collection:'):
            section = config[section_name]
            yield {
                "name": section_name.split(':')[1],
                "workers": section.get('workers', '1'),
            }
```